felix/ipsets/ipsets_test.go

```diff
@@ -284,9 +307,17 @@ var _ = Describe("IP sets dataplane", func() {
 			}
 		})
 
-		It("should clean everything up on first apply()", func() {
+		It("should rate limit clean up", func() {
 			apply()
+			// Should delete one temp and one normal IP set.
```

Comment:
perhaps a note that `MaxIPSetDeletionsPerIteration = 1`

Reply:
I should add a config flag for that, missed that while productising